### ISSUE TITLE
Option to get latest version from the OSF while the experiment is open

### ISSUE THREAD

**Issue by smathot** [dschreij/opensesame-extension-osf#11]
Here's a scenario that I just ran into and that isn't neatly covered yet:

- I was working on an experiment on PC1, and saved it to the OSF.
- I went to PC2 (with an eye tracker), and retrieved the experiment from the OSF.
- On PC2 I made some changes.
- I went back to PC1. Here the experiment was still open, but I wanted to get the latest version from the OSF. Currently, the only way to do that is to close the experiment, and then re-open it so that you can choose whether you want to work on the local version or the OSF version. (Which isn't all that bad, but still.)

So it might be nice to add a button to the explorer and/ or the menu to check whether an update is available. This button should only be visible for linked experiments, of course.

What do you think?

**Comment by dschreij** [comment]
That's a good point. Another option would be to periodically check if there is an update and notify the user if there is (either through a qnotification or a dialog box which you now get when opening an experiment). I think the way you propose might still lead to version conflicts if users forget to check for updates themselves on a regular basis. This automatic regular checks shouldn't be that intensive. Maybe check if the sha numbers are still equal once a minute?

**Comment by smathot** [comment]
> Maybe check if the sha numbers are still equal once a minute?

Yes, that would be even better. I'd give it low priority for now though--I just thought I'd flag it for the future.

I've been using the OSF extension quite a lot lately, and it really works pretty smoothly. I haven't experienced a single serious problem, only little things like this and the gateway error that I reported in the other issue. Great job!

**Comment by dschreij** [comment]
Thanks! Good to hear!
